constraint/pkg/client/drivers/local/local.go

```diff
@@ -45,15 +45,31 @@ func Tracing(enabled bool) Arg {
 	}
 }
 
+func DisableBuiltin(builtin string) Arg {
```

Comment:
Can we make this DisableBuiltins (plural) and allow users to disable multiple builtins?

Reply:
i think this still needs to be changed to plural?